website/docs/docs/use-dbt-semantic-layer/gsheets.md

```diff
@@ -42,32 +42,59 @@ import SLCourses from '/snippets/_sl-course.md';
 
 When querying your data with Google Sheets:
 
-- It returns the data to the cell you have clicked on, and each cell where data is requested will have a note attached to it, indicating what has been queried.
+- It returns the data to the cell you have clicked on, and each cell where data is requested will have a note attached to it, indicating what has been queried and the timestamp.
 - The custom menu operation has a timeout limit of six (6) minutes.
 - If you're using this extension, make sure you're signed into Chrome with the same Google profile you used to set up the Add-On. Log in with one Google profile at a time as using multiple Google profiles at once might cause issues.
 
   
-## Custom menu functions
+## Query Builder functions
 
-The custom menu provides the following capabilities:
+The Google Sheets **Query Builder** custom menu has the following capabilities:
 
 | Menu items    | Description                                           |
 |---------------|-------------------------------------------------------|
 | Metrics       | Search and select metrics.                             |
-| Group By      | Search and select dimensions to group by. Dimensions are grouped by the entity of the semantic model they come from. |
+| Group By      | Search and select dimensions or entities to group by. Dimensions are grouped by the entity of the semantic model they come from. |
 | Time Range    | Quickly select time ranges to look at the data, which applies to the main time series for the metrics (metric time). You can still optionally use `where` to filter time ranges|
 | Granularity   | Modify the granularity of the primary time dimension.      |
 | Where         | Filter your data. This includes categorical and time filters. |
 | Order By      | Return your data order.                              |
 | Limit         | Set a limit for the rows of your output.               |
 
 
-## Filtering data 
+**Filtering data**
 
 To use the filter functionality, choose the [dimension](docs/build/dimensions) you want to filter by and select the operation you want to filter on. 
    - For categorical dimensions, type in the dimension value you want to filter by (no quotes needed) and press enter. 
-   - Continue adding additional filters as needed with AND and OR. If it's a time dimension, choose the operator and select from the calendar. 
+   - Continue adding additional filters as needed with AND and OR. If it's a time dimension, choose the operator and select from the calendar.
 
+**Querying without headers or columns**
+
+If you would like to just query the data values without the headers, you can optionally check the box "Exclude Column Names".
```

Comment:
fyi @rpourzand i dont see this right now in sheets - is that ok? might be helpful to have a screenshot to show what this looks like. non blocker